src/scss/components/layout/_index.scss

```diff
@@ -44,6 +44,7 @@ html {
 
 body {
   background-color: $gray-25;
+  background-color: var(--hxLayout-bgcolor, $gray-25);
```

Comment:
Please update the file to use our new design tokens.  Use the closet available token.